frontend/packages/ceph-storage-plugin/src/components/block-pool/block-pool-list-page.tsx

```diff
@@ -204,6 +205,9 @@ const BlockPoolTableRow: React.FC<RowFunctionArgs<StoragePoolKind>> = ({ obj, cu
           kind={referenceFor(obj)}
           resource={obj}
           isDisabled={disableMenuAction(obj, props?.cephCluster)}
+          terminatingTooltip={
+            isDefaultPool(obj) ? t('ceph-storage-plugin~Default pool cannot be deleted.') : ''
+          }
```

Comment:
```suggestion
          terminatingTooltip={
            isDefaultPool(obj) ? t('ceph-storage-plugin~Default pool cannot be deleted.') : ''
          }
```
This is incorrect we want to show this tooltip even when the resource is not in terminating phase.

Reply:
Okay, so even when the kebab is not disabled?

Reply:
Kebab is always disabled for default pool.

Reply:
ACK.

Reply:
Sent a patch to disable the tooltip for this if block pool is the default one.